docs/tracing.md

```diff
@@ -106,18 +110,19 @@ config:
 
 Client for [Lightstep](https://lightstep.com).
 
-In order to configure Thanos to interact with Lightstep you need to provide at least an [access token](https://docs.lightstep.com/docs/create-and-use-access-tokens) in the configuration file. The `collector` key is optional and used when you have on-premise satellites.
+In order to configure Thanos to interact with Lightstep, you need to provide at least an [access token](https://docs.lightstep.com/docs/create-and-use-access-tokens) in the configuration file. The `collector` key is optional and used when you have on-premise satellites.
+
+[embedmd]: # "flags/config_tracing_lightstep.txt yaml"
 
-[embedmd]:# (flags/config_tracing_lightstep.txt yaml)
 ```yaml
 type: LIGHTSTEP
 config:
   access_token: ""
   collector:
-    scheme: ""
-    host: ""
-    port: 0
-    plaintext: false
-    custom_ca_cert_file: ""
+  scheme: ""
```

Comment:
Is this intended? What changed? Because this should have been autogenerated from the `embedmd` right?